### DISCUSSION TITLE
An issue about HistogramStandardization

### DISCUSSION THREAD

**Discussion by lyangfan** [TorchIO-project/torchio#666] [Q&A]
Hi all:
      I have a question confused me: If I used the HistogramStandardization transform in the training dataset, can I still use the same transform at the inference stage? If used, how to find the landmark?

**Comment by fepegar** [comment]
The landmarks can be stored in a file, for example. Please take a look at the [API](https://torchio.readthedocs.io/transforms/preprocessing.html#torchio.transforms.HistogramStandardization).

**Reply by lyangfan** [reply]
Thanks for your patience!
